src/etc/inc/certs.inc

```diff
@@ -281,6 +281,42 @@ function ca_inter_create(& $ca, $keylen, $lifetime, $dn, $caref, $digest_alg = "
 	return true;
 }
 
+function ca_trustedcerts_generate() {
+	global $config, $g;
+	if (isset($config['system']['developerspew'])) {
+		$mt = microtime();
+		echo "ca_trustedcerts_generate() being called $mt\n";
+	}
+
+	safe_mkdir("{$g['etc_path']}/ssl/certs");
+	/* Remove all certs in the directory (but leave symlinks in place to allow power users to add trusted certs outside the Certificate Manager) */
+	array_map('unlink', array_filter('is_file',glob("{$g['etc_path']}/ssl/certs/*")));
```

Comment:
Believe array_filter syntax is incorrect.  The callback function should be the second parameter.
Even so in my testing is_file was true for symlinks too.  So don't think this will do what is expected.